[Bug]: ./test.sh failing with errors

### NetBox access-list plugin version

v1.6.1

### NetBox version

v4.0.11

### Steps to Reproduce

 Run ./test.sh

### Expected Behavior

Tests Complete as expected

### Observed Behavior

Exception occurs - 

test_root (netbox_acls.tests.test_api.AppTest.test_root) ... ok

======================================================================
FAIL: test_graphql_get_object (netbox_acls.tests.test_api.ACLTestCase.test_graphql_get_object)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/netbox/venv/lib/python3.11/site-packages/django/test/utils.py", line 443, in inner
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/netbox/netbox/utilities/testing/api.py", line 555, in test_graphql_get_object
    self.assertIn('errors', data)
AssertionError: 'errors' not found in {'data': {'access_list': {'changelog': [], 'display': 'testacl1', 'class_type': 'AccessList', 'tags': [], 'custom_fields': {}, 'assigned_object_type': {'id': '36'}, 'id': '1', 'created': '2024-09-12T21:47:09.050112+00:00', 'last_updated': '2024-09-12T21:47:09.050119+00:00', 'custom_field_data': {}, 'name': 'testacl1', 'assigned_object_id': 1, 'type': 'standard', 'default_action': 'deny', 'comments': ''}}}

----------------------------------------------------------------------
Ran 22 tests in 0.493s

FAILED (failures=1, skipped=1)

--- a/netbox_acls/graphql/__init__.py
+++ b/netbox_acls/graphql/__init__.py
@@ -2,8 +2,6 @@ from .schema import *
 from .types import *
 
 schema = [
-    schema.NetBoxACLSAccessListQuery,
-    schema.NetBoxACLSStandardRuleQuery,
-    schema.NetBoxACLSACLExtendedRuleQuery
+    schema.NetBoxACLSQuery
 ]
 

--- a/netbox_acls/graphql/schema.py
+++ b/netbox_acls/graphql/schema.py
@@ -2,31 +2,19 @@ import strawberry
 import strawberry_django
 from .types import *
 from ..models import *
+from typing import List
 
-@strawberry.type
-class NetBoxACLSAccessListQuery:
+@strawberry.type(name="Query")
+class NetBoxACLSQuery:
     """
     Defines the queries available to this plugin via the graphql api.
     """
-    @strawberry.field
-    def access_list(self, id: int) -> AccessListType:
-        return AccessList.objects.get(pk=id)
-    access_list_list: list[AccessListType] = strawberry_django.field()
+    access_list: AccessListType = strawberry_django.field()
+    access_list_list: List[AccessListType] = strawberry_django.field()
 
-@strawberry.type
-class NetBoxACLSACLExtendedRuleQuery:
-    @strawberry.field
-    def acl_extended_rule(self, id: int) -> ACLExtendedRuleType:
-        return ACLExtendedRule.objects.get(pk=id)
-    acl_extended_rule_list: list[ACLExtendedRuleType] = strawberry_django.field()
+    acl_extended_rule: ACLExtendedRuleType = strawberry_django.field()
+    acl_extended_rule_list: List[ACLExtendedRuleType] = strawberry_django.field()
 
-
-
-@strawberry.type
-class NetBoxACLSStandardRuleQuery:
-    @strawberry.field
-    def acl_standard_rule(self, id: int) -> ACLStandardRuleType:
-        return ACLStandardRule.objects.get(pk=id)
-    acl_standard_rule_list: list[ACLStandardRuleType] = strawberry_django.field()
-
+    acl_standard_rule: ACLStandardRuleType = strawberry_django.field()
+    acl_standard_rule_list: List[ACLStandardRuleType] = strawberry_django.field()
 

